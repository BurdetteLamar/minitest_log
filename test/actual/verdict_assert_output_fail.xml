<log>
  <summary verdicts='2' failures='1' errors='0'/>
  <verdict method='verdict_assert_output?' outcome='failed' id='1'>
    <stdout>
      foo
    </stdout>
    <stderr>
      bar
    </stderr>
    <exception class='Minitest::Assertion'>
      <message>
        In stderr. --- expected +++ actual @@ -1,2 +1 @@ -# encoding: UTF-8
        -&quot;bar&quot; +&quot;foo&quot;
      </message>
      <backtrace/>
    </exception>
  </verdict>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
