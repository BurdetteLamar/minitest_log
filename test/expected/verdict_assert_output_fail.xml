<log>
  <summary_ verdicts='1' failures='1' errors='0'/>
  <verdict_ method='verdict_assert_output?' outcome='failed' id='1'>
    <stdout_ class='String' value='&quot;foo&quot;'/>
    <stderr_ class='String' value='&quot;bar&quot;'/>
    <exception_ class='Minitest::Assertion' message='In stderr.'>
      <backtrace_/>
    </exception_>
  </verdict_>
</log>
